src/Twig/Extension/StringExtension.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Twig\Extension;
+
+use Sonata\AdminBundle\Twig\Extension\UnicodeString as DecoratedUnicodeString;
+use Symfony\Component\String\UnicodeString;
+use Twig\Extension\AbstractExtension;
+use Twig\Extra\String\StringExtension as TwigStringExtension;
+use Twig\TwigFilter;
+
+/**
+ * Decorates `Twig\Extra\String\StringExtension` in order to provide the `$cut`
+ * argument for `Symfony\Component\String\UnicodeString::truncate()`.
+ * This class must be removed when the component ships this feature.
+ *
+ * @internal
+ *
+ * @see https://github.com/symfony/symfony/pull/35649
+ *
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+final class StringExtension extends AbstractExtension
```

Comment:
Tests please

Reply:
I think this is already tested [here](https://github.com/sonata-project/SonataAdminBundle/pull/5983/files#diff-47b1548a4d062a7a56630ceb30fc1bbcR207).